object-group-game/Location/Location.cs

```diff
@@ -6,17 +6,26 @@
 
 namespace object_group_game
 {
-    class Location
-    {
-        private float Latitude { get; set; }
-        private float Longtitude { get; set; }
-        private LocationType LocationType { get; set; }
+	public class Location
+	{
+		public int ID { get; set; }
+		public string DisplayName { get; set; }
+		public LocationType Type { get; set; }
+		public double Latitude { get; set; }
+		public double Longtitude { get; set; }
+		public int Radius { get; set; }
+		public Item Item { get; set; }
 
-        public Location(float latitude, float longtitude, LocationType locationType)
-        {
-            this.Latitude = latitude;
-            this.Longtitude = longtitude;
-            this.LocationType = locationType;
-        }
-    }
+
+		public Location (Database.Location location, Item item)
+		{
+			ID = location.ID;
+			DisplayName = location.DisplayName;
+			Type = location.Type;
+			Latitude = location.Latitude;
+			Longtitude = location.Longtitude;
+			Radius = location.Radius;
+			Item = item;
+		}
+	}
```

Comment:
More properties are good, but get rid of any references to Database here

Reply:
Again it is convenient to create this class this way. Don't see an issue.